cirq-google/cirq_google/__init__.py

```diff
@@ -174,7 +175,8 @@
         ),
         'FSIM_GATESET': (
             'v0.16',
-            'SerializableGateSet will no longer be supported.'
+            'SerializableGateSet and associated classes (GateOpSerializer, GateOpDeserializer,'
```

Comment:
This message is getting pretty long.  Should we make it a constant?

Reply:
Ack, I added two identical constants to avoid circular dependencies.